docs/sources/operations/zone-ingesters.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Zone Aware Ingesters
+description: Migration guide and additional details
```

Comment:
This is the social media/Google search preview text.  Could we be more specific about what we're migrating to help users know if this is the link they're searching for?

Reply:
How much detail should we put? Are search results not the combination of the title and the description?

`Zone aware ingester migration guide and additional details`?

Reply:
Sadly I recently learned that our internal search on the Grafana docs site displays ONLY the description metadata in the search results.  This is another place where I need to take a pass through the documentation and add some helpful content.

![GL_Search](https://github.com/grafana/loki/assets/4106682/829677b9-fafa-47d3-8500-734e4275e8f4)

Reply:
Ugh, sorry, that screen grab turned out to be huge.  The second line under the bold heading is pulled from the `description` in the metadata.